common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -90,6 +90,30 @@
     ],
     "sqlState" : "42000"
   },
+  "APPROX_TOP_K_MAX_ITEMS_TRACKED_EXCEEDS_LIMIT": {
+    "message": [
+      "The max items tracked `maxItemsTracked` = <maxItemsTracked> of `approx_top_k` should be less than or equal to <limit>."
+    ],
+    "sqlState" : "22023"
+  },
+  "APPROX_TOP_K_MAX_ITEMS_TRACKED_LESS_THAN_K": {
+    "message": [
+      "The max items tracked `maxItemsTracked` = <maxItemsTracked> of `approx_top_k` should be greater than or equal to `k` = <k>."
```

Comment:
nit:
"The max items tracked `maxItemsTracked`(<maxItemsTracked>) of `approx_top_k` should be greater than or equal to `k`(<k>)."